lang/en.json

```diff
@@ -782,6 +782,7 @@
         "TriggerCannotBeDeleted": "This trigger cannot be deleted",
         "TriggerConditionNode": "Note: Some variables may not work as you expect. For example using a click variable in combination with a form variable might only work if you have also added a click trigger to your container. This applies typically to variables within the categories Clicks, Forms, History, Errors, Scrolls, and Visibility which should are typically used in their respecting trigger only.",
         "TriggerConditionsHelp": "You can further restrict when this trigger should be triggered by specifying one or multiple conditions. Only when ALL of these conditions are met, the trigger will be actually triggered (not when only one of them is met). Only trigger when all of these conditions are met:",
+        "TriggerConditionsHelpText": "%1$sLearn more%2$s about the click classes condition.",
```

Comment:
I know this text was decided on over a year ago, but could we update it to better match the content of the linked article? Something like this:

%1$sLearn more%2$s about tracking interactions on nested elements.

Reply:
Updated as suggested